pkg/db/db.go

```diff
@@ -0,0 +1,138 @@
+//Provides operations on SQLite DB
+package db
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+//Database type is sql.DB
+type Database sql.DB
+
+//Record type represents DB record description
+type Record struct {
+	ID, Name, Host, dataType, Data string
+}
+
+func (db *Database) sql() *sql.DB {
+	return (*sql.DB)(db)
+}
+
+//Connect connects to DB with given name database name
+func Connect(db_name string) (*Database, error) {
+	conn, err := sql.Open("sqlite3", db_name)
+	if err != nil {
+		return nil, err
+	}
+	db := (*Database)(conn)
+	if err := db.createTable(); err != nil {
+		db.Close()
+		return nil, err
+	}
+	return db, nil
+}
+
+//Close closes the connection to the database
+func (db *Database) Close() error {
+	return db.sql().Close()
+}
+
+func (db *Database) createTable() error {
+	sql := `
+	  CREATE TABLE IF NOT EXISTS migrations(
```

Comment:
Take this as my personal opinion and suggestion.
I don't like to execute DDL inside app, I understand that structure is small and it's not a big deal, although it's bad to keep DDL and DML in one.